plasmapy/particles/ionization_state.py

```diff
@@ -912,4 +912,5 @@ def summarize(self, minimum_ionic_fraction: Real = 0.01) -> None:
             output += attributes
 
         for line in output:
-            print(line)
+            # We'll need to switch from print() to using logging library
```

Comment:
```suggestion
```
I *think* that since we're intentionally printing this to the console, that `print` should be fine here.  If any issues arise, we can change it.